website/docusaurus.config.ts

```diff
@@ -464,6 +464,17 @@ export default async function createConfigAsync() {
           } satisfies BlogOptions,
           pages: {
             remarkPlugins: [npm2yarn],
+            editUrl: ({locale, pagesPath}) => {
+              if (locale !== defaultLocale) {
+                return `https://crowdin.com/project/docusaurus-v2/${locale}`;
+              }
+              // We want users to submit updates to the upstream/next version!
+              // Otherwise we risk losing the update on the next release.
+              const nextVersionDocsDirPath = 'src/pages';
+              return `https://github.com/facebook/docusaurus/edit/main/website/${nextVersionDocsDirPath}/${pagesPath}`;
```

Comment:
```suggestion
              // We want users to submit updates to the upstream/next version!
              // Otherwise we risk losing the update on the next release.
              const nextVersionDocsDirPath = 'src/pages';
              return `https://github.com/facebook/docusaurus/edit/main/website/src/pages/${pagesPath}`;
```

The comment only makes sense for docs